src/main/java/com/spider/solomon/order/CustomerOrderDao.java

```diff
@@ -34,7 +36,13 @@ public CustomerOrder getOrderById (int orderId) {
 	 */
 	public List<CustomerOrder> getOrdersByCustomerId (int customerId) {
 		String sql = "SELECT * FROM customer_orders WHERE customer_id = ?";
-		return jdbcTemplate.query(sql, ORDER_MAPPER, customerId);
+
+		List<CustomerOrder> ordrs = jdbcTemplate.query(sql, ORDER_MAPPER, customerId);
+		ordrs.forEach(order -> {
```

Comment:
This uses N+1 queries, which would be very inefficient for large orders. What do you think about updating the SQL to use a join so that we can do it all in a single query?

[Meta: I'm certain this is enough detail for you to make the correction. If a more junior developer submitted this, I might give more specific feedback depending on their skill level. For example, I might explain in more detail how each call to `getOrderItemsByOrderId` fires an additional query, explain how to modify the SQL to include a `LEFT JOIN`, give detail on how to update the row mapping, or maybe point to an example elsewhere in the codebase]